### PR TITLE
React Dividers

### PR DESCRIPTION
**Pull request by corcorankt1**
## What does this PR do?
Add SprkDivider component for React.

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR
then please remove it.
### Documentation
 - [x] Update Spark Docs React

### Code
 - [x] Build Component in Spark React
 - [x] Unit Testing in Spark React with `gulp test-react` (100% coverage, 100% passing)

### Accessibility
- [x] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [x] Apple Safari
  - [ ] Apple Safari (Mobile)

### Design Review
 - [ ] Design reviewed and approved

### Screenshots
Add screenshots to help explain your PR if you'd like. However, this is not
expected.

### CODE REVIEW

**Comment by afebbraro** [comment]
@corcorankt1 I think we want to give the ability to use a span or a <hr> here. Could you update this so that they can pick which element they want. Could check out rob's button PR to see how he did it. Nice job otherwise!!!

File: src/patterns/components/dividers/react/info/base.hbs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+  <thead>
+  <tr>
+    <th>Input</th>
```
Comment (afebbraro) [inline src/patterns/components/dividers/react/info/base.hbs]: Input should be Prop.

File: src/patterns/components/dividers/react/info/base.hbs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <tbody>
+  <tr>
+    <td class="sprk-u-FontWeight--bold">element</td>
+    <td>string</td>
+    <td>Determines the element that will be rendered. Options are "hr" or "span". This is required.</td>
+  </tr>
+  <tr>
+    <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+    <td>string</td>
+    <td>Allows a string of classes to add, in addition to the spark classes. Intended for overrides.</td>
```
Comment (afebbraro) [inline src/patterns/components/dividers/react/info/base.hbs]: spark should be Spark

File: src/react/src/routes/Router.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Homepage from './Homepage/Homepage';
 import SparkButtonDocs from './SparkButtonDocs/SparkButtonDocs';
 import SprkLinkDocs from './SprkLinkDocs/SprkLinkDocs';
+import SprkDividers from './SprkDividersDocs/SprkDividersDocs';
```
Comment (bobwise) [inline src/react/src/routes/Router.js]: I think these should be called "SprkDivider" (singular) and "SprkDividerDocs" to match the other component names.

File: src/patterns/components/dividers/react/info/base.hbs (hunk 2 above)
Comment (corcorankt1) [reply src/patterns/components/dividers/react/info/base.hbs]: Updated both!

File: src/react/src/routes/Router.js (hunk 3 above)
Comment (corcorankt1) [reply src/react/src/routes/Router.js]: Updated!

**Pull request merged**